### PR TITLE
Add Billing Usage Prometheus System Component

### PR DESCRIPTION
**Pull request by jpweber**
This PR's primary reason is to add the manifests to deploy the prometheus servers for use for billing purposes only. 

Since this is the first of a new pattern please take time to digest how I went about implementing this and freely suggest changes. 

I also updated the circleci file to only try to deploy this to dev for now so we don't break any other environments until we know this all works. Once we know it all works in dev correctly I'll update the file to deploy to stage and prod.

### CODE REVIEW

**Comment by jpweber** [comment]
Work for https://github.com/astronomer/issues/issues/1432

File: Dockerfile [hunk 1]
```diff
@@ -39,3 +40,7 @@ RUN helm3 plugin install https://github.com/helm/helm-2to3.git
 RUN pip3 install --upgrade setuptools pip
 RUN apk add alpine-sdk libffi-dev
 RUN pip3 install google-cloud-storage pyyaml
+
+RUN curl -L -s -O "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2F${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz" && \
```
Comment (danielhoherd) [inline Dockerfile:44]: Can we delete the downloaded file here? Also while we're at it we should apply that same optimization to the previous downloads that don't have it.

**Review by danielhoherd — APPROVED** [review]
Looks rad. The Kustomize stuff is easy to understand, and it looks like a good tool to solve our needs.

File: Dockerfile (hunk 1 above)
Comment (jpweber) [reply Dockerfile:44]: Good call. Yup I'll add it anywhere it isn't already while in there.

**Pull request merged**